notebooks/official/pipelines/automl_tabular_classification_beans.ipynb

```diff
@@ -2,13 +2,13 @@
   "cells": [
```

Comment:
<p>I believe there is a GCPC component now for AutoML model eval</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
True (ModelEvaluationOp), but there are no new predictions being generated in this notebook. The evaluation metrics from the trained AutoML model are logged and used for conditioning the model deployment.